backend/src/middleware/middleware.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface RequestBody
```

Comment:
Please rename to Register

There should be a Login model in the related login challenge middleware aswell